bunq/sdk/json/adapters.py

```diff
@@ -29,25 +29,25 @@ class InstallationAdapter(converter.JsonAdapter):
     @classmethod
     def deserialize(cls, target_class, array):
         """
-        :type target_class: model.Installation|type
+        :type target_class: core.Installation|type
```

Comment:
Is this notation used every where else in this SDK as well ? 🙊

Reply:
Notation, as in `|type`? Yes, absolutely! For long time already :)
Same goes for `module.Class`! 👍